sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -294,6 +302,14 @@ export class KeyClient {
     return this.getKeyFromKeyBundle(response);
   }
 
+  /**
+   * Parses the KeyVault unique identifier of a KeyVault Key.
+   * @param identifier The unique identifier of the KeyVault key
+   */
+  public parseIdentifier(identifier: string): ParsedKeyVaultIdentifier {
```

Comment:
Should this be static? And if so, I don't see much value in putting it on clients. Maybe just a `KeyVaultIdentifier.parse` method. It's all in one place then.

Reply:
I mean, if it's not static, how did you construct a client without a proper URI to begin with? The ask was to pass a full URI and be able to get a client for it.

Reply:
I'm not understanding the question. I'll ping you over Teams.

Reply:
I asked through teams and you have indeed provided the answers I needed! Thank you!